Frameworks/CoreGraphics/CGContext.mm

```diff
@@ -2138,8 +2174,9 @@ void CGContextClearRect(CGContextRef context, CGRect rect) {
     ComPtr<ID2D1CommandList> commandList;
     RETURN_IF_FAILED(deviceContext->CreateCommandList(&commandList));
 
-    deviceContext->BeginDraw();
+    EscapeBeginEndDrawStack();
```

Comment:
Going to do this in a separate PR instead

---
In reply to: [96997659](https://github.com/Microsoft/WinObjC/pull/1750#discussion_r96997659) [](ancestors = 96997659)